spec/services/platform_apis/cautionary_alerts/client_spec.rb

```diff
@@ -2,18 +2,26 @@
 
 require "rails_helper"
 
-RSpec.describe PlatformApis::CautionaryAlerts::Client do
+RSpec.describe PlatformApis::Alerts::Client do
   let(:request) { double }
 
   before do
     allow(described_class).to receive(:request).and_return(request)
   end
 
-  describe ".get_cautionary_alerts_by_property_reference" do
+  describe ".get_alerts_by_property_reference" do
     it "calls the Request object with a property reference id" do
       expect(request).to receive(:retrieve).with("cautionary-alerts/properties/100023022310")
 
-      described_class.get_cautionary_alerts_by_property_reference("100023022310")
+      described_class.get_alerts_by_property_reference("100023022310")
+    end
+  end
+
+  describe ".get_alerts_by_tenancy_reference" do
+    it "calls the Request object with a property reference id" do
+      expect(request).to receive(:retrieve).with("cautionary-alerts/people?tag_ref=01111/01")
+
+      described_class.get_alerts_by_tenancy_reference("01111/01")
     end
   end
 end
```

Comment:
you forgot to rename the file_name `spec/services/platform_apis/cautionary_alerts/client_spec.rb` to `spec/services/platform_apis/alerts/client_spec.rb`